include/RAJA/real_datatypes.hxx

```diff
@@ -111,10 +111,10 @@ typedef std::complex<Real_type> Complex_type;
 // alignment attribute supported for versions > 12
 //
 #if __ICC >= 1300
```

Comment:
We can probably get rid of this check. However, we still have v10 Intel compilers installed on our platforms !?!?